pypdf/_writer.py

```diff
@@ -795,11 +812,14 @@ def update_page_form_field_values(
         for j in range(len(page[PG.ANNOTS])):  # type: ignore
             writer_annot = page[PG.ANNOTS][j].get_object()  # type: ignore
             # retrieve parent field values, if present
-            writer_parent_annot = {}  # fallback if it's not there
+            writer_parent_annot = cast(DictionaryObject, {})  # fallback if it's not there
```

Comment:
Is there a disadvantage in just creating that object?

```suggestion
            writer_parent_annot = DictionaryObject()  # fallback if it's not there
```